src/lib/components/filter/_filter.scss

```diff
@@ -14,7 +14,8 @@
     }
 
     .terra-filter-actions {
-        padding: 0 var(--space-lg);
+        padding: var(--space-md) var(--space-lg) 0 var(--space-lg);
+        border-top: 1px solid var(--color-structure-3);
```

Comment:
I think it's better if you remove the border-top property and use a <mat-divider> inside ShowcaseFilterComponent in terra repo. But @zakpur or @vwiebe can decide if it's better.

Reply:
and maybe `padding: var(--space-md) var(--space-md) 0 0` ?

Reply:
![image](https://user-images.githubusercontent.com/89769969/140313598-a52a8aaa-ab92-4604-8e2d-437fc25cfe3c.png)

Reply:
KK, I will ask regarding the mat-divider. 
I think the padding is fine as it is, but Saman or Viktor should approve that too. :)

Reply:
The problem with the divider would be that everyone would need to include it by themselves.
With a general border style, it would apply to all instances of the filter-menu automatically.